### ISSUE TITLE
Somfy sunis 

### ISSUE THREAD

**Issue by Zlugger44** [totalimmersion/com.somfy.tahoma#34]
Hey!

Thanks for a great app!

After the upgrade to 2.0, solar censors have stopped reporting values.

Thanks!
Mikael

**Comment by FPPMvB** [comment]
Hi, same issue here, I have not yet tested the other Somfy parts but this one is no longer useable in flows

**Comment by totalimmersion** [comment]
I've tested it myself as well and on my Homey the Sunis sensors still report values. Could you describe your setup (ie. which version of Homey, which version of the Tahoma app). And could you describe how I can reproduce the issue. With just 'it doesn't work anymore' I can't help to solve the issue.

**Comment by Marcelklim** [comment]
Hello Lennart,

These are the settings i have. 







Success. If you have more questions you can ofcourse mail me.

Greetings,


Marcel Ubels

**Comment by Zlugger44** [comment]
Hi!

Problem persists, no values in Somfy TaHoma app in Homey.

Homey model: early 2018
Homey version: 2.0.0
Homey app version: 2.0.2.246
Somfy TaHoma app: v1.5.1

![img_2448](https://user-images.githubusercontent.com/46840215/51537728-a1f5c900-1e4f-11e9-8c65-eb797666581a.PNG)
![img_2447](https://user-images.githubusercontent.com/46840215/51537748-b043e500-1e4f-11e9-97fc-8b258abb7735.PNG)

Thanks!
Mikael

**Comment by totalimmersion** [comment]
Thank you for the info. Could you also try to remove the device from Homey
and then add it again and see if it still doesn't report any values?

Best,

Lennart

**Comment by Zlugger44** [comment]
Has just removed the device and added it, the result is the same - no values.

thanks for your effort!
Mikael

**Comment by Marcelklim** [comment]
Have removed the app and the devices and installed it again. Everything is working now, the sunsensor io and the suncreen io.

**Comment by keolse** [comment]
Hello

Are there any information you are missing for this fault?

**Comment by keolse** [comment]
Hello, i got the sensors working now, adjusted sync speed. Then they where working. Or have you fixed issue in last update? :)

**Comment by totalimmersion** [comment]
No, I did not fix it in the last update. I am still trying to figure out what's causing the bug to appear for some users and not for others. Could you tell me what your sync speed values were before you adjusted it and what it is now?

**Comment by keolse** [comment]
Hello. It's not he sync that is the problem. Seems to be something with the  IO vertical exterior blinds.

When i add this to homey data to sun sensors stoppes. When i delete it. Data transfer starts again.

**Comment by frankveldink** [comment]
OK, same problem here and found a way to reproduce the issue:

SITUATION
- Device: Somfy Sunis WireFree II IO 
- Somfy native iOS app Connexoon: works and Sunis values are registering
- Homey: Somfy app configured and added the Sunis device - works and Sunis values are registering in Homey
- Homey: Added a vertical sunscreen - works and Sunis values are registering in Homey
- Homey: Added a horizontal sunscreen - registration of Sunis values in Homey STOPS and gets stuck on the latest succesful reading

TROUBLESHOOTING
- Step 1 Homey: Somfy app syncspeed adjusted from 10 sec, to 5 sec, to 15 sec - no effect
- Step 2 Homey: Removed horizontal sunscreen - Sunis values are immediately registrating again (without removing the vertical sunscreen)
- Step 3 Homey: Added the horizontal sunscreen again - registration of Sunis values STOPS and gets stuck on the latest succesful reading again
- Step 4 Homey: Removed horizontal sunscreen again - Sunis values are immediately registrating again (without removing the vertical sunscreen)

EXTRA INFO:
- The vertical sunscreen is a light motor
- The horizontal sunscreen is a heavy motor
- The Somfy iOS app differentiates those 2 motors, perhabs related to the Sunis Homey app?

**Comment by Zlugger44** [comment]
As previously noted, the problem with Exterior horizontal awning (io) causes Sunis sun sensor to lose contact with Homey.
When this is installed, measurement values will cease.
However, works well with Exterior vertical blind (io).

**Comment by frankveldink** [comment]
@totalimmersion can you give is some information on your progression?

**Comment by jmoorman777** [comment]
Hi

For me it works sonetimes
But very often i get a value in the app and than when i click the value it says “ 2 days ago “

If i reset the homey it works sgain for a while

Somfy itself doesnt show this problem

**Comment by stefan-b-b** [comment]
Same issue. It does not give me a reading for the sun sensor. Please help.

**Comment by stefan-b-b** [comment]
> Same issue. It does not give me a reading for the sun sensor. Please help.

Note: I have removed the IO devices from my Homey and now it is working. I am now able to write flows and at least control the IO Devices by means of triggering Tahoma Scenarios. If I want to control the blinds I need to use Google home but most IO devices are supposed to work on timers and scenarios so that is an interim solution.

**Comment by wieooit** [comment]
My Sunis has never worked in Homey.
It only works in the Tahoma box

Early 2018 Homey
Tahoma box
3 io screens and 5 io blinds
